mlflow/models/utils.py

```diff
@@ -264,3 +263,17 @@ def plot_lines(data_series, xlabel, ylabel, legend_loc=None, line_kwargs=None):
     ax.set(xlabel=xlabel, ylabel=ylabel)
 
     return fig, ax
+
+
+def create_wheeled_model(artifact_path, model_uri):
+    """
+    Given a registered model, this helper will re-log the model along with the wheels of all the
+    the model dependencies (stored along with the model artifacts).
+
+    :param artifact_path: Run-relative artifact path.
+    :param model_uri: registered model uri of the form
+                        models:/<model_name>/<model_version/stage/latest>
```

Comment:
Can we include a Python code example demonstrating how this API can be used to:

1. Add libraries to an existing model stored in MLflow Tracking
2. Re-log a registered model version with extra libraries

?